checks/__init__.py

```diff
@@ -265,6 +278,12 @@ def get_metrics(self, expire=True):
                 pass
         return metrics
 
+    def get_stats(self):
```

Comment:
Do we need it here? This class is a legacy class, not sure it's worth any further development.